packages/main/src/themes/StepInput.css

```diff
@@ -182,14 +186,14 @@
 }
 
 :host([value-state=Error]:not([readonly]):not([disabled]))::after {
-    border-color: var(--sapField_InvalidColor);
+	border-color: var(--sapField_InvalidColor);
 }
 
 /* Disable spin buttons in Chrome, Safari, Edge, Opera */
 :host .ui5-step-input-input::-webkit-outer-spin-button,
 :host .ui5-step-input-input::-webkit-inner-spin-button {
-  -webkit-appearance: none;
-  margin: 0;
+	-webkit-appearance: none;
+	margin: 0;
 }
 
 :host([disabled]) {
```

Comment:
When the component is disabled, it should have opacity added over the regular state. Which means that we should have the same background image as in regular state and opacity on top

Reply:
done